backend/coreapp/serializers.py

```diff
@@ -48,18 +47,14 @@ def serialize_profile(
             "is_online": profile.is_online(),
             "is_admin": user.is_staff,
             "username": user.username,
-            "avatar_url": github_details.avatar_url if github_details else None,
         }
 
         if small:
             return small_obj
 
         return {
             **small_obj,
-            "email": user.email,
-            "name": github_details.name if github_details else user.username,
-            "github_api_url": github_details.url if github_details else None,
-            "github_html_url": github_details.html_url if github_details else None,
+            "github_id": gh_user.github_id if gh_user else None,
```

Comment:
It probs makes sense to get rid of the small/large user distinction in future if there's only one extra field